azure-cron.yml

```diff
@@ -1,213 +0,0 @@
-# Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
-# SPDX-License-Identifier: Apache-2.0
-
-# Azure Pipelines file, see https://aka.ms/yaml
-
-# Do not run on PRs
-pr: none
-
-# Do not run on merge to main
-trigger: none
-
-# Do run on a schedule (hourly)
-#
-# This is currently (2019-08-15) broken on Azure for GitHub-hosted repos. It
-# does, however, work as expected for Azure-hosted repos. As a workaround, we
-# have created a repo inside Azure that contains an `azure-pipelines.yml` file
-# that just triggers this job.
-#
-# When the situation is resolved, delete that repo in Azure and uncomment the
-# following. In the meantime, this should stay commented so we avoid running
-# jobs twice when Azure fixes this issue.
-#schedules:
-#- cron: "0 * * * *"
-#  displayName: hourly cron
-#  branches:
-#    include:
-#    - main
-#  always: true
-
-jobs:
-  - job: fix_bazel_cache
-    timeoutInMinutes: 120
-    pool:
-      name: 'ubuntu_20_04'
-      demands: assignment -equals default
-    steps:
-      - checkout: self
-      - bash: ci/dev-env-install.sh
-        displayName: 'Build/Install the Developer Environment'
-      - template: ci/bash-lib.yml
-        parameters:
-          var_name: bash-lib
-      - bash: |
-          set -euo pipefail
-          eval "$(dev-env/bin/dade assist)"
-
-          bazel build //ci/cron:cron
```

Comment:
We can possibly remove that too from the main-2.x branch. Not sure what that might affect. I'll try.